packages/core/src/connectors/injected.ts

```diff
@@ -146,9 +146,9 @@ export function injected(parameters: InjectedParameters = {}) {
       const provider = await this.getProvider()
       if (!provider) throw new ProviderNotFoundError()
 
-      let accounts: readonly Address[] | null = null
-      if (!isReconnecting) {
-        accounts = await this.getAccounts().catch(() => null)
+      let accounts: readonly Address[] = []
+      if (isReconnecting) {
```

Comment:
Is this meant to be changed?

Reply:
This is where the actual change is. `!isRexonnecting` to `isRexonnecting`. The rest of the typescript specific type related stuff was to get lint a d build to work.

Reply:
I refactored so only the necessary changes are in the PR. This line is really the only line which needs changing.